src/OrchardCore.Modules/OrchardCore.Layers/Drivers/LayerMetadataWelder.cs

```diff
@@ -10,51 +11,45 @@
 
 namespace OrchardCore.Layers.Drivers;
 
-public class LayerMetadataWelder : ContentDisplayDriver
+public sealed class LayerMetadataWelder : ContentDisplayDriver
 {
     private readonly ILayerService _layerService;
-    protected readonly IStringLocalizer S;
 
-    public LayerMetadataWelder(ILayerService layerService, IStringLocalizer<LayerMetadataWelder> stringLocalizer)
+    internal readonly IStringLocalizer S;
+
+    public LayerMetadataWelder(
+        ILayerService layerService,
+        IStringLocalizer<LayerMetadataWelder> stringLocalizer)
     {
         _layerService = layerService;
         S = stringLocalizer;
     }
 
-    protected override void BuildPrefix(ContentItem model, string htmlFieldPrefix)
-    {
-        base.BuildPrefix(model, htmlFieldPrefix);
-        if (string.IsNullOrWhiteSpace(htmlFieldPrefix))
-        {
-            Prefix = "LayerMetadata";
```

Comment:
Do does it need to be LayerMetadata?

Reply:
I don't think it specifically needs to be that (apart from cases where somebody might have hard-coded this already, including in JS or CSS), but something unique, yes. In any case, unless there's some evidence for why this isn't good, this shouldn't be changed.

Reply:
I don't think there is a reason to override the Prefix in this case. We don't do this anywhere else and we should not do it here. It simplifies the code and aligns it with everything else we do. Prefix should be computed as we do everything else.

Reply:
E.g part drivers set it from the type of the part. This, however, doesn't, and `ContentItem` as a prefix (because `ContentDisplayDriver` is a driver for any content item) is too generic to be unique. This opens up clashing form field names. Setting the prefix explicitly is thus appropriate.

Reply:
okay. I changed this back